tests/test_config.py

```diff
@@ -544,3 +544,22 @@ def test_bind_fd_works_with_reload_or_workers(reload, workers):  # pragma: py-wi
     assert sock.getsockname() == ""
     sock.close()
     fdsock.close()
+
+
+@pytest.mark.parametrize(
+    "reload, workers, expected",
+    [
+        (True, 1, True),
+        (False, 2, True),
+        (False, 1, False),
+    ],
+    ids=[
+        "--reload=True --workers=1",
+        "--reload=False --workers=2",
+        "--reload=False --workers=1",
+    ],
+)
+def test_config_use_subprocess(reload, workers, expected):
+    config = Config(app=asgi_app, reload=reload, workers=workers)
+    config.load()
+    assert config.use_subprocess == expected
```

Comment:
`use_process` doesn't exist anymore on this branch. I was fine with it tho 🤔

Reply:
ok I reverted too much, goinmg to do that tomorrow sorry :_)

Reply:
mixed up things etc... tired

Reply:
No problem! Have a good evening man! ♥️🙏

Reply:
ok should be ok now @Kludex , apologies for the confusion :)